.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2021.05.0rc4
+current_version = 2021.06.0rc0
```

Comment:
this doesn't match the other version numbers (this one has a `rc0`, the others not)

Reply:
You mean we started with rc1 in the other releases?

Reply:
No, here you define the version to be `2021.06.0rc0`, but below this version is linked: https://github.com/raiden-network/raiden-service-bundle/tree/2021.06.0

Reply:
Or will the image name have the `rc0` removed?

Reply:
Argh, no bumpversion again 🙈 
Thanks for noticing

Reply:
Fixed